apps/central-scan/frontend/src/screens/scan_ballots_screen.tsx

```diff
@@ -102,32 +112,29 @@ export function ScanBallotsScreen({
             disabled={isScanning || statusIsStale}
             isScannerAttached={status.isScannerAttached}
           />
-          <Button
-            onPress={() => setIsExportingCvrs(true)}
-            disabled={
-              status.adjudicationsRemaining > 0 || status.batches.length === 0
-            }
-            nonAccessibleTitle={exportButtonTitle}
-            icon="Export"
-            color="primary"
-          >
-            Save CVRs
-          </Button>
+          <ActionsLeft>
```

Comment:
thoughts?

<img width="1163" alt="Screen Shot 2024-10-11 at 11 07 35 AM" src="https://github.com/user-attachments/assets/0e7feadd-11ba-4a4b-bdde-cd9f49a48569">
<img width="1153" alt="Screen Shot 2024-10-11 at 12 36 34 PM" src="https://github.com/user-attachments/assets/4b1a4dd7-3e9d-4749-9f1c-f1d7d8fa7d34">

Reply:
I think having the text floating between the buttons looks a tad weird. What about this?

![Screenshot-VxCentralScan-2024-10-14T22:12:07 010Z](https://github.com/user-attachments/assets/4ddf58bc-8641-473e-932a-793e174cbedd)

Idea is to move the batch and ballot totals near the relevant table columns and group the buttons (I also bumped up the size of the totals so they would take up a bit more space and look less weirdly floaty, though they are still a bit untethered)

Reply:
@jonahkagan I think that's an improvement. What do you think of using a callout here to make it a little less floaty and tie together the summary data? 

<img width="1151" alt="Screen Shot 2024-10-14 at 4 13 14 PM" src="https://github.com/user-attachments/assets/275529d5-cb36-44a4-9d98-9bb3e2a4fa58">

Reply:
The callout is just a tad taller than the buttons, which looks weird to me. But making it button sized would maybe make it look like a button... Hmmm. I dunno

Maybe we leave it floaty but move everything a bit closer to the table?

Reply:
Can you get with the smaller callout?

<img width="1151" alt="Screen Shot 2024-10-14 at 4 46 42 PM" src="https://github.com/user-attachments/assets/bad64ea7-a671-467c-b08a-089bc68e9611">
<img width="1152" alt="Screen Shot 2024-10-14 at 4 49 53 PM" src="https://github.com/user-attachments/assets/17635dd1-8941-4231-9712-6cf1241d6fd7">

Reply:
Haha sure

Reply:
Did you check the empty case with this new layout? Does that need to change?

Reply:
yeah, will grab a video

Reply:
It looked weird in a callout so I just left it as is, and moved it to the same corner:

https://github.com/user-attachments/assets/a7fa212b-2d51-4fc8-8997-67ebaed7772c

Reply:
@mattroe can you offer a fresh pair of eyes on this particular change? At this point it all looks strange to me.

Reply:
@adghayes what am I looking for exactly? the batch count/sheet count change? I think that's a great move. The adjudication options have always been a bit confusing, but I think we're doing the best we can there.

Reply:
I'm talking about the re-arrangement of the buttons and the counts in this thread.

Reply:
Didn't notice any button re-arrangement tbh and I like the way you're showing the sheet vs. batch count. Ship it